src/effects/DynamicRangeProcessorHistoryPanel.cpp

```diff
@@ -37,8 +37,18 @@ static const wxColor inputColor { 71, 154, 42, 128 };
 static const wxColor outputColor { 103, 124, 228, 128 };
 static const wxColor actualCompressionColor { 255, 255, 255 };
 
-constexpr auto actualCompressionLineWidth =
-   DynamicRangeProcessorPanel::targetCompressionLineWidth + 4;
+bool MayUsePenGradients()
+{
+   // MacOS doesn't cope well with pen gradients. (Freezes in debug and is
```

Comment:
nitpick: you can write "MacOS and Linux doesn't cope..." as this function checks if the current platform is running Windows (only) -  just to to reduce potential questions in the future

Reply:
Agreed. Will solve when rebasing.